Source/WebCore/Modules/webauthn/AuthenticatorCoordinator.cpp

```diff
@@ -191,6 +191,17 @@ void AuthenticatorCoordinator::create(const Document& document, CredentialCreati
         ASSERT(!exception.message.isNull());
         promise.reject(exception.toException());
     };
+
+    if (m_isCancelling) {
+        m_queuedRequest = [weakThis = WeakPtr { *this }, weakFrame = WeakPtr { *frame }, weakDocument = WeakPtr<Document, WeakPtrImplWithEventTargetData> { document }, clientDataJsonHash = WTFMove(clientDataJsonHash), createOptions = WTFMove(createOptions), callback = WTFMove(callback)]() mutable {
```

Comment:
I think we should take in `document->securityOrigin() `(`callerOrigin`) here instead of getting in inside the lambda.